lib/iris/quickplot.py

```diff
@@ -89,12 +90,16 @@ def _label(cube, mode, result=None, ndims=2, coords=None, axes=None):
         raise ValueError(msg)
 
 
-def _label_with_bounds(cube, result=None, ndims=2, coords=None, axes=None):
-    _label(cube, iris.coords.BOUND_MODE, result, ndims, coords, axes)
+def _label_with_bounds(
+    cube, result=None, ndims=2, coords=None, axes=None, colorbar=True
+):
+    _label(cube, iris.coords.BOUND_MODE, result, ndims, coords, axes, colorbar)
```

Comment:
As well as `pcolor()` and `pcolormesh()`, this is also called by the `outline()` function. Is there a reason why you didn't add the `colorbar` parameter there as well?

Reply:
See comment above regarding `points` method.